cmd/krel/cmd/release_notes.go

```diff
@@ -264,17 +264,13 @@ func createDraftPR(tag string, result *releaseNotesResult) error {
 		}
 	}
 
-	// verify the branch doesn't already exist on the user's fork
-	err = sigReleaseRepo.HasRemoteBranch(branchname)
-	if err == nil {
-		return errors.Errorf("remote repo already has a branch named %s", branchname)
-	}
-
 	// checkout the new branch
-	err = sigReleaseRepo.Checkout("-b", branchname)
-	if err != nil {
+	if err = sigReleaseRepo.Branch(branchname, "--force"); err != nil {
```

Comment:
Do you think we could also do `git checkout -B <branch>`? I think this should behave in the same way.

Reply:
I forgot about `-B`. 🤦‍♂

Reply:
It looks like we've got some rework with the rest of the logic, so I'll update this when I make those changes.